payjoin-cli/README.md

```diff
@@ -1,111 +1,187 @@
+nigiri rpc generatetoaddress $(nigiri rpc getnewaddress "sender") 101
+nigiri rpc generatetoaddress $(nigiri rpc getnewaddress "receiver") 101
 ```
 
-where `$VERSION` is the [latest version](https://crates.io/crates/payjoin-cli) of the `payjoin-cli` you wish to install.
+Great! Our wallets are setup, now let's do an async payjoin.
 
-Get a list of commands and options:
+### Install `payjoin-cli`
 
-```console
-payjoin-cli --help
+```sh
+cargo install payjoin-cli --version $VERSION --features v2
 ```
 
-Either pass config options from cli, or manually edit a `config.toml` file within directory you run `payjoin-cli` from.
-Configure it like so:
+where `$VERSION` is the [latest version](https://crates.io/crates/payjoin-cli).
+
+Next, create a directory for the sender & receiver and create a `config.toml` file for each:
+
+```sh
+mkdir sender receiver
+touch sender/config.toml receiver/config.toml
+```
+
+Edit the `config.toml` files. Note that the `v2` feature requires a payjoin directory server and OHTTP relay. Learn more about them [here](https://payjoin.org/docs/how-it-works/payjoin-v2-bip-77).
 
 ```toml
-# config.toml
-bitcoind_cookie = "/tmp/regtest1/bitcoind/regtest/.cookie"
-# specify your wallet via rpchost connection string
-bitcoind_rpchost = "http://localhost:18443/wallet/boom"
- ```
+# sender/config.toml
 
-Your configuration details will vary, but you may use this as a template.
+# Nigiri uses the following RPC credentials
+bitcoind_rpcuser = "admin1"
+bitcoind_rpcpassword = "123"
+bitcoind_rpchost = "http://localhost:18443/wallet/sender"
 
-## Test Payjoin 2
+# For v2, our config also requires a payjoin directory server and OHTTP relay
+pj_directory = "https://payjo.in"
+ohttp_relay = "https://pj.bobspacebkk.com"
+```
 
-### Install `payjoin-cli` with the V2 feature
+```toml
+# receiver/config.toml
 
-```console
-cargo install payjoin-cli --version $VERSION --features v2
+# Nigiri uses the following RPC credentials
+bitcoind_rpcuser = "admin1"
+bitcoind_rpcpassword = "123"
+bitcoind_rpchost = "http://localhost:18443/wallet/receiver"
+
+# For v2, our config also requires a payjoin directory server and OHTTP relay
+pj_directory = "https://payjo.in"
+ohttp_relay = "https://pj.bobspacebkk.com"
 ```
 
-### V2 Configuration
+Now, the receiver must generate an address to receive the payment:
 
-In addition to the rpc configuration above, specify relevant ohttp and payjoin directory configuration as follows:
+```sh
+receiver/payjoin-cli receive 10000
+```
 
-```toml
-# config.toml
-...
-# a production payjoin directory server
-pj_directory="https://payjo.in"
-# payjo.in's ohttp_keys can now be fetched rather than configured ahead of time
- # an ohttp relay with ingress to payjo.in
-ohttp_relay="https://pj.bobspacebkk.com"
+This will output a [BIP21](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki) URL containing the receiver's address, amount, payjoin directory, and OHTTP relay. For example:
```

Comment:
- URI != URL
- I prefer the name of the thing over the BIP number if referencing the thing it defines and not the spec itself
- OHTTP relay isn't encoded. Only client session info (session keys, ohttp key configuration for directory, expiration time encoded in the fragment)
```suggestion
This will output a [bitcoin URI](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki) containing the receiver's address, amount, payjoin directory, and other session information the client needs. For example:
```